WordPress/Classes/ViewRelated/Reader/ReaderCommentsViewController.m

```diff
@@ -1171,6 +1189,57 @@ - (void)presentWebViewControllerWithURL:(NSURL *)URL
 
 #pragma mark - PostHeaderView helpers
 
+- (void)handleFollowingConversationButtonTapped
+{
+    __typeof(self) __weak weakSelf = self;
+
+    UINotificationFeedbackGenerator *generator = [UINotificationFeedbackGenerator new];
+    [generator prepare];
+
+    // Keep previous subscription status in case of failure
+    BOOL oldIsSubscribed = self.postHeaderView.isSubscribedToPost;
+    BOOL newIsSubscribed = !oldIsSubscribed;
+
+    // Optimistically toggle subscription status
+    self.postHeaderView.isSubscribedToPost = newIsSubscribed;
+
+    // Define success block
+    void (^successBlock)(void) = ^void() {
+        NSString *subscriptionString = newIsSubscribed
+            ? NSLocalizedString(@"subscribed to", "Past tense verb + preposition. A subscription action.")
+            : NSLocalizedString(@"unsubscribed from", "Past tense verb + preposition. A subscription action.");
+        NSString *successTitle = [NSString stringWithFormat: NSLocalizedString(@"Successfully %@ the post", @"The app successfully updated the subscription status for the post"), subscriptionString];
```

Comment:
What do you think about creating the whole string at once?

I fear that this might be a little bit confusing to the translators and that we might have issues with how this sentence might be written in other languages.

Reply:
That sounds like a much simpler idea, thanks!